website/src/content/api-reference/deployment-previews.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Deployment previews
+---
+
+<Subtitle>
+
+Deployment previews let you test your Custom Application features before publishing them to the production environment.
+
+</Subtitle>
+
+This document describes the necessary steps to build and add deployment previews to your Custom Application in the Merchant Center.
```

Comment:
Nit: the steps to add deployment previews belong more to the MC docs. I would focus here only on the application config setup.

Reply:
That's true. But, to complete the loop, we have mentioned the link to MC docs as the last step of this process.